### DISCUSSION TITLE
Gist Search Box Text Unreadable (White on White)

### DISCUSSION THREAD

**Discussion by marketinview** [community/community#39951] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Recently, I noticed that when I type in the Gist search box I can't see what I am typing because it is white text on a white background while focused on the search box (i.e., .header-search-input:focus).

**Comment by Akash1134** [comment]
Hi @marketinview , Could you please share a screenshot illustrating the issue? I am not able to reproduce this issue at my end. Thanks!

**Comment by marketinview** [comment]
![image](https://user-images.githubusercontent.com/11151876/203832463-12466073-4c54-4a87-8e26-bf49ecd60d49.png)
